homework_11/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+from unittest.mock import patch
+
+@pytest.fixture
+def mock_get_logger():
+    with patch('homework_11.log_event.logging.getLogger') as mock_logger:
```

Comment:
It is no need to mock logger function. We need to test real logger actions into file writing. In this case it is better to parse real file with logs and get real messages from there and compare with expected results.